src/Features/CSharp/Portable/EditAndContinue/SyntaxComparer.cs

```diff
@@ -377,7 +383,14 @@ private static Label ClassifyStatementSyntax(SyntaxKind kind, SyntaxNode? node,
                     return Label.ForEachStatement;
 
                 case SyntaxKind.UsingStatement:
-                    return Label.UsingStatement;
+                    // We need to distinguish using statements with no or single variable declaration from one with multiple variable declarations. 
```

Comment:
Am I reading it wrong, or does the comment not match the code? The code looks like it's distinguishing between no declarations, from single or multiple declarations. Shouldn't it check count of declarations?

Reply:
"For simplicity we do not match single-declaration to expression, we just treat usings
                    // with declarations entirely separately from usings with expressions."

Reply:
Meaning - using with expression needs to be different from using with multiple declarations. We might match it with using with a single variable declaration, but it gets complicated and it's not necessary.